checkov/common/sca/reachability/typing.py

```diff
@@ -0,0 +1,4 @@
+from typing import List, Dict
+
+# {<package_name>: {"packageAliases": [<alias_1> ,..... , <alias_n>]}}
+FileParserOutput = Dict[str, Dict[str, List[str]]]
```

Comment:
Let's create stricter types for the entire mapping object (can leverage pydantic for that)
Since we will pass this object to sast-core we need to have a clear view of the "contract"

Reply:
agreed. let's talk !